### ISSUE TITLE
null pointer exception at startup

### ISSUE THREAD

**Issue by leginee** [ceylon/ceylon-ide-intellij#572]
Hello,
I got following null pointer exception at startup.
```
null
java.lang.NullPointerException
	at org.intellij.plugins.ceylon.ide.project.PageOne.apply(PageOne.java:93)
	at org.intellij.plugins.ceylon.ide.project.CeylonModuleBuilder.persistConfiguration(CeylonModuleBuilder.java:100)
	at org.intellij.plugins.ceylon.ide.project.CeylonModuleBuilder$1.moduleCreated(CeylonModuleBuilder.java:41)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.intellij.util.EventDispatcher.dispatch(EventDispatcher.java:99)
	at com.intellij.util.EventDispatcher.access$200(EventDispatcher.java:35)
	at com.intellij.util.EventDispatcher$2.invoke(EventDispatcher.java:79)
	at com.sun.proxy.$Proxy67.moduleCreated(Unknown Source)
	at com.intellij.ide.util.projectWizard.ModuleBuilder.onModuleInitialized(ModuleBuilder.java:273)
	at com.intellij.ide.util.projectWizard.ModuleBuilder.access$000(ModuleBuilder.java:56)
	at com.intellij.ide.util.projectWizard.ModuleBuilder$2.lambda$run$0(ModuleBuilder.java:302)
	at com.intellij.openapi.application.impl.ApplicationImpl.runWriteAction(ApplicationImpl.java:898)
	at com.intellij.ide.util.projectWizard.ModuleBuilder$2.run(ModuleBuilder.java:302)
	at com.intellij.ide.startup.impl.StartupManagerImpl.runActivity(StartupManagerImpl.java:342)
	at com.intellij.ide.startup.impl.StartupManagerImpl.runActivities(StartupManagerImpl.java:334)
	at com.intellij.ide.startup.impl.StartupManagerImpl.runPostStartupActivities(StartupManagerImpl.java:175)
	at com.intellij.openapi.project.impl.ProjectManagerImpl.lambda$null$4(ProjectManagerImpl.java:343)
	at com.intellij.openapi.application.TransactionGuardImpl$6.run(TransactionGuardImpl.java:291)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.runNextEvent(LaterInvocator.java:326)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.run(LaterInvocator.java:310)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:756)
	at java.awt.EventQueue.access$500(EventQueue.java:97)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.awt.EventQueue$3.run(EventQueue.java:703)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:726)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:793)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:629)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:385)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)

```
Steps to reproduce:
1) start IntellJ
2) new Project
3) select JVM (->default; 1.8) as SDK
4) confirm all other options.

I am on Arch haveing intellJ just installed following packages:
intellij-idea-libs-2:2016.2.4-1  intellij-idea-community-edition-2:2016.2.4-1

and then installed the plugin through plugin... -> repository -> ceylon IDE

All the best
Peter

**Comment by leginee** [comment]
update: After Project has been created, at next startup the null pointer exception does not show.

**Comment by bjansen** [comment]
Hi, and thanks for you interest in this plugin :). That error is already fixed on master, but I forgot to integrate the fix in the stable branch. 

1.3.1 should be released pretty soon, so I'll mark this issue as fixed in 1.3.1. If you want to try nightly builds (which already contain the fix), you can follow the [procedure described in the README](https://github.com/ceylon/ceylon-ide-intellij#install-from-the-development-plugin-repository).

**Comment by bjansen** [comment]
Linked to #547
